opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectContent.jsx

```diff
@@ -217,11 +251,23 @@ class StixDomainObjectContentComponent extends Component {
     this.subscriptionToggle = MESSAGING$.toggleNav.subscribe({
       next: () => this.setState({ navOpen: localStorage.getItem('navOpen') === 'true' }),
     });
-    this.loadFileContent();
+    this.subscription = interval$.subscribe(() => {
+      this.props.relay.refetch();
```

Comment:
so this component will refetch stixDomainObject query with importFiles, exportFiles, and externalReferences every 5 seconds, always. It doesn't feel right for me since we just want to display exported pdf files.

Reply:
Didn't for me neither at first, @Kedae told us that wasn't so bad to fetch all the object

Reply:
I wonder if it would be possible to refetch only if there is a file export in progress, otherwise, no refetch.

Reply:
Oh, that was a discussion we also had. I don't remember why we didn't do it @CelineSebe ?
Indeed it could be nice to stop refetching when the exported file is ready

Reply:
Done